tensorflow_model_optimization/python/core/clustering/keras/cluster_distributed_test.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Distributed clustering test."""
+
+from absl.testing import parameterized
+import numpy as np
+import tensorflow as tf
+
+from tensorflow_model_optimization.python.core.keras import test_utils as keras_test_utils
+from tensorflow_model_optimization.python.core.clustering.keras import cluster
+from tensorflow_model_optimization.python.core.clustering.keras import cluster_config
+from tensorflow_model_optimization.python.core.clustering.keras import cluster_wrapper
+
+keras = tf.keras
+CentroidInitialization = cluster_config.CentroidInitialization
+
+
+def _distribution_strategies():
+  return [
+      # tf.distribute.experimental.MultiWorkerMirroredStrategy(),
```

Comment:
Please, leave in only the strategies we want to be testing here, and remove unnecessary commented code here and in other places below.

Reply:
Thanks, done.